src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -153,6 +153,13 @@ private bool TryGetTextForSymbol(SyntaxToken token, SemanticModel semanticModel,
                 symbol = ((ILocalSymbol)symbol).Type;
             }
 
+            // Range variable: use the type
+            if (symbol is IRangeVariableSymbol)
+            {
+                var info = semanticModel.GetSpeculativeTypeInfo(token.SpanStart, SyntaxFactory.ParseName(symbol.Name), SpeculativeBindingOption.BindAsExpression);
+                symbol = info.Type;
```

Comment:
@gafter Can you comment on this? @amcasey mentioned there were some complexities to determining the type of an IRangeVariableSymbol that resulted in the type not being available through the interface.